builtin/mainmenu/generate_from_settingtypes.lua

```diff
@@ -31,7 +31,7 @@ local group_format_template = [[
 #    octaves     = %s,
 #    persistence = %s,
 #    lacunarity  = %s,
-#    flags       = %s
+#    flags       =%s
```

Comment:
This is a bit ugly, but I can't think of a better way to do it that doesn't involve more substantial changes.